site/cypress/e2e/Cypress-Gradeable/rainbow_grading.spec.js

```diff
@@ -4,18 +4,166 @@ skipOn(Cypress.env('run_area') === 'CI', () => {
         beforeEach(() => {
             cy.login('instructor');
             cy.visit(['sample', 'config']);
+            checkCheckbox('[data-testid="display-rainbow-grades-summary"]');
+            cy.visit(['sample', 'reports', 'rainbow_grades_customization']);
+            cy.wait(1000);
```

Comment:
Is this wait needed? (if it is can you add a comment why)

Reply:
```suggestion
            cy.get('#list_wrapper').should('be.visible');
```
This should work in place of a static wait.

Reply:
Removed wait() in [f49f89d](https://github.com/Submitty/Submitty/pull/10726/commits/f49f89d5f0cf92dfdb4ba38b50bbad550aa354fa)